Game/Game.pde

```diff
@@ -15,17 +19,23 @@ void setup() {
   //noCursor();
 }
 
+void createGameObjects() {
```

Comment:
What do you think about moving this function to a separate file? It's going to grow a lot.

Reply:
Ok. a new file or to an existing one? What would you name it? If you know, you can already implement it :)

Reply:
I'll do it

Reply:
I don't think I can change your code though, do you know if I can do it?